Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabFocusHandler.cpp

```diff
@@ -14,38 +14,51 @@
 #include <AzToolsFramework/Entity/ReadOnly/ReadOnlyEntityInterface.h>
 #include <AzToolsFramework/Prefab/Instance/Instance.h>
 #include <AzToolsFramework/Prefab/Instance/InstanceEntityMapperInterface.h>
+#include <AzToolsFramework/Prefab/Instance/InstanceUpdateExecutorInterface.h>
 #include <AzToolsFramework/Prefab/PrefabDomUtils.h>
 #include <AzToolsFramework/Prefab/PrefabFocusNotificationBus.h>
 #include <AzToolsFramework/Prefab/PrefabFocusUndo.h>
 #include <AzToolsFramework/Prefab/PrefabSystemComponentInterface.h>
 
 namespace AzToolsFramework::Prefab
 {
-    PrefabFocusHandler::PrefabFocusHandler()
+    void PrefabFocusHandler::RegisterPrefabFocusInterface()
```

Comment:
This function does way more than just `RegisterPrefabFocusInterface` - maybe it could be renamed to `Initialize`?

Reply:
After taking a look at other prefab interfaces, I suggest keeping the naming. Other interfaces do more or less other stuff in register method as well. IMO being able to quickly notice that the interfaces will be registered in the function as the name suggests helps me understand better.